dotcom-rendering/cdk/bin/cdk.ts

```diff
@@ -44,20 +45,28 @@ new RenderingCDKStack(cdkApp, 'ArticleRendering-PROD', {
 	scaling: {
 		minimumInstances: 18,
 		maximumInstances: 90,
-		policy: {
-			scalingStepsOut: [
-				// No scaling up effect when latency is lower than 0.2s
-				{ lower: 0, upper: 0.2, change: 0 },
-				// When latency is higher than 0.2s we scale up by 50%
-				{ lower: 0.2, change: 50 },
-				// When latency is higher than 0.3s we scale up by 80%
-				{ lower: 0.3, change: 80 },
-			],
-			scalingStepsIn: [
-				// No scaling down effect when latency is higher than 0.12s
-				{ lower: 0.12, change: 0 },
-				// When latency is lower than 0.12s we scale down by 1
-				{ upper: 0.12, lower: 0, change: -1 },
+		policies: {
+			step: {
+				scalingStepsOut: [
+					// No scaling up effect when latency is lower than 0.2s
+					{ lower: 0, upper: 0.2, change: 0 },
+					// When latency is higher than 0.2s we scale up by 50%
+					{ lower: 0.2, change: 50 },
+					// When latency is higher than 0.3s we scale up by 80%
+					{ lower: 0.3, change: 80 },
+				],
+				scalingStepsIn: [
+					// No scaling down effect when latency is higher than 0.12s
+					{ lower: 0.12, change: 0 },
+					// When latency is lower than 0.12s we scale down by 1
+					{ upper: 0.12, lower: 0, change: -1 },
+				],
+			},
+			target: [
+				{
+					type: PredefinedMetric.ASG_AVERAGE_CPU_UTILIZATION,
```

Comment:
Could there be a comment explaining what's going on here, similar to the above which is really useful. I assume this means that when hit the average CPU utilisation of 40% we scale up by 1?

Reply:
Also a bit of a shame that [we can only use average](https://github.com/aws/aws-cdk/blob/8ebfadeb8d7a8c15176ed9e5f106a49fc84040a6/packages/aws-cdk-lib/aws-autoscaling/lib/target-tracking-scaling-policy.ts#L163-L167), and not something like P90 or P99